altair/utils/mimebundle.py

```diff
@@ -102,17 +108,29 @@ def _spec_to_mimebundle_with_engine(spec, format, mode, **kwargs):
         # from SCHEMA_VERSION (of the form 'v5.2.0')
         vl_version = "_".join(SCHEMA_VERSION.split(".")[:2])
         if format == "vega":
-            vg = vlc.vegalite_to_vega(spec, vl_version=vl_version)
+            if mode == "vega":
+                vg = spec
+            else:
+                vg = vlc.vegalite_to_vega(spec, vl_version=vl_version)
             return {"application/vnd.vega.v5+json": vg}
         elif format == "svg":
-            svg = vlc.vegalite_to_svg(spec, vl_version=vl_version)
+            if mode == "vega":
+                svg = vlc.vega_to_svg(spec)
```

Comment:
Is it intentional that the `vega_to_*` functions do not need to take the Vega version as an argument? I'd assume that once Vega 6 would come out, Altair and VLC need to be kept in sync anyway by a user and so they would both be upgraded at the same time and hence `vlc` would pick the correct Vega version again by default. Just want to check if that matches your understanding.

Reply:
Makes sense, thanks!